events/sink.go

```diff
@@ -0,0 +1,59 @@
+package events
+
+import (
+	"context"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/containerd/containerd/api/types/event"
+	"github.com/containerd/containerd/namespaces"
+	goevents "github.com/docker/go-events"
+	"github.com/pkg/errors"
+)
+
+type sinkEvent struct {
+	ctx   context.Context
+	event Event
+}
+
+type eventSink struct {
```

Comment:
We have to take another pass here when filters are integrated. Could we fix this at that time?